examples/Forms/TextInput.md

```diff
@@ -110,6 +109,48 @@ const [value, setValue] = useState('Some valid input');
 />
 ```
 
+Label example: The optional `errorMessage` prop is not set, and the 
+`hideError` prop is set to `true.` Remove the `errorMesage` style.
+
+```jsx
+import { useState } from 'react';
+
+const [value, setValue] = useState('Some valid input');
+
+<TextInput
+  value={value}
+  name="example"
+  id="example"
+  label="Description:"
+  hideError={true}
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Label example: The optional `errorMessage` prop is set, and the 
+`hideError` prop is set to `true.` Remove the `errorMesage` and
+its style.
```

Comment:
Typo: `errorMessage`
I would also change this to "`hideError` prop is set to `true,` which removes the `errorMessage` and its style."